content/post/bonne-annee-2023.en.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Happy new year 2023"
+date: 2023-01-14
+draft: false
+---
+
+Here we are, our planet just finished another revolution around its star and started a new one.
+
+In all our names, please allow me to wish you a good and happy new year 2023.
```

Comment:
On everyone's/everybody's behalf instead?

Reply:
Changed ("on behalf of all of us")